src/site/content/en/blog/layout-instability-api/index.md

```diff
@@ -0,0 +1,331 @@
+destinations on the web. With the addition of CLS as of the May 2019 release,
+developers can get a better understanding of how users experience layout
+instability on their sites, on their competitors' sites, and on the web as a
+whole.
+
+<div class="w-aside w-aside--objective">
+  <p><strong>Important:</strong>
+  while most sites should strive for a CLS score of 0, it's certainly
+  possible that some sites employ layout shifts deliberately  (for example, in
+  multimedia presentations, progressive visualisations, slideshows, etc.).</p>
+  <p>
+  This is perfectly fine. CLS scores are intended to help developers who may
+  not be aware of the problems caused by unexpected layout shifts; they're not
+  intended to suggest that sites that deliberately shift the layout are
+  necessarily problematic.</p>
+</div>
+
+## How to use the Layout Instability API
+
+The Layout Instability API is available in Chrome 74+ with the experimental web
+platform features flag enabled
+(`chrome://flags/#enable-experimental-web-platform-features`) or by through the
+[Layout Instability Origin
+Trial](https://developers.chrome.com/origintrials/#/view_trial/1215971899390033921),
+which will be available until Sept. 3, 2019.
+
+Similar to other performance APIs, Layout Instability can be observed via the
+`PerformanceObserver` interface, where you can subscribe to entries of type
+`layoutShift`.
+
+The following code logs all layout shift entries as they happen:
+
+```js
+const observer = new PerformanceObserver((list) => {
+  for (const entry of list.getEntries()) {
+    console.log(entry);
+  }
+})
+
+observer.observe({entryTypes: ['layoutShift']});
+```
+
+If you want to calculate the cumulative layout shift score for your pages and
+track them in your analytics back end, you can declare a variable that stores
+the current cumulative layout shift score, and then increment it any time a new
+layout shift is detected. You'll typically want to record scores from the
+initial page load until the page's lifecycle state changes to hidden:
+
+```js
+// Stores the current layout shift score for the page.
+let cumulativeLayoutShiftScore = 0;
+
+// Detects new layout shift occurrences and updates the
+// `cumulativeLayoutShiftScore` variable.
+const observer = new PerformanceObserver((list) => {
+  for (const entry of list.getEntries()) {
+    cumulativeLayoutShiftScore += entry.value;
+  }
+});
+
+observer.observe({entryTypes: ['layoutShift']});
+
+// Sends the final score to your analytics back end once
+// the page's lifecycle state becomes hidden.
+document.addEventListener('visibilitychange', () => {
+  if (document.visibilityState === 'hidden') {
+    // Force any pending records to be dispatched.
+    observer.takeRecords();
+
+    // Send the final score to your analytics back end
+    // (assumes `sendToAnalytics` is defined elsewhere).
+    sendToAnalytics({cumulativeLayoutShiftScore});
+  }
+});
+```
+
+<div class="w-aside w-aside--note">
```

Comment:
We have shortcodes you can use for this. Sorry if it's not well documented.

```
{% Aside 'note' %}
[you can put markdown in here]
{% endAside %}
```

Reply:
I wanted to customize what was bolded, and it didn't seem like I could do that with the short codes. Let me know if I'm wrong.